activities.html

```diff
@@ -7,6 +7,7 @@ <h1 class="small">Activities</h1>
     <div class="jumbotron split">
         <img class="inline-main-img" src="/images/activities-main.png">
         <p>Welcoming Falls Church is building a community of neighbors. How can we do this? Some of this is cultural. We can always smile, prioritize relationships, engage with newcomers (immigrants and otherwise), and bring people together. Culture is contagious, and a welcoming culture brings life.</p>
+        <p><b>Coming soon:</b> Alan Gratz joins Welcoming Falls Church on October 28, 2020, for a virtual discussion of his New York Times bestselling novel, Refugee. This award-winning book follows three children at three different points in time, all connected by the same goal: escape. <a href="mailto:welcomingfallschurch@gmail.com">Email us</a> if you would like to join us at 4-5pm for the online program - and we'll send you event login information, and see <a href="http://welcomingfallschurch.org/activities/learn/refugee">here</a> for some great resources that we have brought together, to support and inspire young readers of Refugee!</p>
```

Comment:
Just one thing, when creating links from one page to another on the same site, you can just do href="/activities/learn/refugee" without the welcomingfallschurch.org. No need to change it now, they both work.

Reply:
Got it. Thanks for letting me know.

Reply:
Definitely do fix this in the next PR though, best to stick to standards when possible.